hub/management/commands/import_wildlife_trust_regions.py

```diff
@@ -0,0 +1,43 @@
+from django.conf import settings
+
+import pandas as pd
+
+from hub.models import DataSet
+
+from .base_importers import BaseImportFromDataFrameCommand
+
+
+class Command(BaseImportFromDataFrameCommand):
+    help = "Import Wildlife trust reserves"
+
+    data_file = settings.BASE_DIR / "data" / "wildlife_trust_regions.csv"
+    cons_row = "Constituency"
+    uses_gss = False
+    message = "Importing constituency Wildlife Trust regions data"
+
+    data_sets = {
+        "wildlife_trust_regions": {
+            "defaults": {
+                "label": "Wildlife Trust Regions",
+                "description": "The Wildlife Trust Region(s) that a constituency falls under.",
+                "data_type": "text",
+                "category": "place",
+                "source_label": "Data from The Widlife Trusts",
+                "source": "https://www.wildlifetrusts.org/",
+                "source_type": "csv",
+                "table": "areadata",
+                "is_filterable": True,
+                "comparators": DataSet.in_comparators(),
```

Comment:
missing unit_type and unit_distribution

Reply:
This shouldn't have gotten committed at all, and was a mistake!